trainingaddon/acceleratoraddon/web/src/de/hybris/training/controllers/cms/CategoryProductsCarouselComponentController.java

```diff
@@ -17,6 +17,7 @@
 @RequestMapping("/view/" + CategoryProductsCarouselComponentModel._TYPECODE+ "Controller")
 public class CategoryProductsCarouselComponentController extends AbstractCMSAddOnComponentController<CategoryProductsCarouselComponentModel> {
 
+    final int TEN_PRODUCTS = 10;
 
```

Comment:
this should be hardcoded, add an attribute to the comonent max products to display and validate, in that way that info would be configurable from backoffice